lib/apartment.rb

```diff
@@ -0,0 +1,16 @@
+class Apartment
+
+attr_reader :number, :monthly_rent, :bathrooms, :bedrooms, :renter
+  def initialize(attributes)
+    @number = attributes[:number]
+    @monthly_rent = attributes[:monthly_rent]
+    @bathrooms = attributes[:bathrooms]
+    @bedrooms = attributes[:bedrooms]
+    @renter = []
+  end
+
+  def add_renter(renter)
+    @renter << renter
```

Comment:
An apartment can only hold one renter and the renter method should return an instance of renter, not an array that holds one renter.